bin/assigner/src/main.cpp

```diff
@@ -74,6 +77,11 @@ int curve_dependent_main(const std::string& input_block_file_name,
         std::cerr << "Assigner run failed: " << run_err.value() << std::endl;
         return 1;
     }
+
+    /*if (!check_assignment_tables()) {
```

Comment:
Commented code

Reply:
That is an example where should be called check assignment tables functions